pythonwhois/net.py

```diff
@@ -2,6 +2,21 @@
 from codecs import encode, decode
 from . import shared
 
+
+WHOIS_REQUEST_FORMAT_EXCEPTIONS = {
```

Comment:
This isn't consistent with the rest of the codebase - it'd have to be lower_snake_case (eg. `whois_request_format_exceptions`).